### ISSUE TITLE
Receiver page not accepting files

### ISSUE THREAD

**Issue by markmacgillivray** [OAButton/discussion#1120]
And when URLs are provided, not showing warnings if they are from places we won't accept, like researchgate

**Comment by markmacgillivray** [comment]
I have tested this, the URLs thing was just too specifically set, it expected researchgate.com rather than just researchgate, and they use researchgate.net, so I have changed and tested that, it works now.

For file upload that seems to be working fine, but it won't successfully submit to zenodo unless there is a valid author name to use. It was originally intended that receiver pages could only be accessed by authors so by that point a name would be available. We now have it that admins can provide a name so I have updated the receiver to use the admin-provided name if nothing else is available, and I have tested this and it works.

However, note it is now currently possible that if someone goes to a receiver page and submits something there for a request that we don't yet have any author name or email address for, the system will accept the submission but then zenodo will refuse it, so the upload will be lost. We either have to block receiver pages from working if there is no name to use, or use some default name like "Anon Author"

Once the above is decided, I can implement and close this issue.
